src/partials/50_section-products.html

```diff
@@ -25,6 +25,9 @@
       <div class="col-3">
         <div class="product-box">
           <div class="photo">
+            <img
+              src="https://urzadzaj24.pl/3227-large_default/kanapa-pikowana-doria-.jpg"
+            />
```

Comment:
img to znacznik niezamykalny - ostatni / zbędny

Reply:
racja, przypadakiem mi się to wkradło...

Reply:
@DawMas usuń proszę ostatni `/` w elemencie img, zgodnie z komentarzem, czyli zamiast `<img />` => `<img >`

Reply:
Poprawione, mój edytor kodu chyba mi to wstawia domyślnie...